tests/camping_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import camping
+from utils.camping_argparser import CampingArgParser
+
+
+class CampingTest(unittest.TestCase):
+
+    def testAggregatesDataForMultipleCampsites(self):
+        park_info = {'1': [],
+                     '2': ['2022-06-22T00:00:00Z', '2022-06-23T00:00:00Z', '2022-06-26T00:00:00Z'],
+                     '3': ['2022-06-22T00:00:00Z', '2022-06-23T00:00:00Z', '2022-06-26T00:00:00Z',
+                               '2022-06-27T00:00:00Z', '2022-06-29T00:00:00Z']}
+        _, _, available_dates_by_campsite_id = \
```

Comment:
Does black actually do this, or did you add the backslash yourself?

Reply:
I believe I added this one.

Reply:
Could we remove it and let black do its thing instead?

Reply:
Yeah! Will remove. I'll rerun black one more time as well just in case anything else got missed.